log.go

```diff
@@ -119,6 +120,7 @@ func init() {
 	chanbackup.UseLogger(chbuLog)
 	monitoring.UseLogger(promLog)
 	wtclient.UseLogger(wtclLog)
+	peernotifier.UseLogger(peerLog)
```

Comment:
You need to create a new subsystem logger above, with tag "PRNF". If you look at the `subsystemsLoggers` map below, there should be a 1:1 mapping between tags and loggers :)

Reply:
Ok cool, I was wondering about that - `ntfnLog` threw me off a bit because it's used twice for `NTFN` and `NTFR`? Typo maybe?

Reply:
Ooh, yeah that's a typo. I put up a fix: https://github.com/lightningnetwork/lnd/pull/3374

Good catch! 😅